README.md

```diff
@@ -1,17 +1,301 @@
 # pack - Buildpack CLI [![Build Status](https://travis-ci.org/buildpack/pack.svg?branch=master)](https://travis-ci.org/buildpack/pack)
 
-**pack** is a tool to create runnable images from applications using buildpacks.
+**`pack`** makes it easy for
+- **application developers** to use [buildpacks](https://buildpacks.io/) to convert code into runnable images
+- **buildpack authors** to develop and package buildpacks for distribution
 
-For information on buildpacks: [buildpacks.io](https://buildpacks.io/)
+## Contents
+- [Building App Images Using `build`](#building-app-images-using-build)
+  - [Example: Building Using the Default Builder Image](#example-building-using-the-default-builder-image)
+  - [Example: Building Using a Specified Buildpack](#example-building-using-a-specified-buildpack)
+  - [Building Explained](#building-explained)
+- [Updating App Images Using `rebase`](#updating-app-images-using-rebase)
+  - [Example: Rebasing an App Image](#example-rebasing-an-app-image)
+  - [Rebasing Explained](#rebasing-explained)
+- [Working with Builders Using `create-builder`](#working-with-builders-using-create-builder)
+  - [Example: Creating a Builder from Buildpacks](#example-creating-a-builder-from-buildpacks)
+  - [Builders Explained](#builders-explained)
+- [Managing Stacks](#managing-stacks)
+  - [Example: Adding a Stack](#example-adding-a-stack)
+  - [Example: Updating a Stack](#example-updating-a-stack)
+  - [Example: Deleting a Stack](#example-deleting-a-stack)
+  - [Example: Setting the Default Stack](#example-setting-the-default-stack)
+  - [Listing Stacks](#listing-stacks)
+- [Resources](#resources)
+- [Development](#development)
 
-## Example Usage
+----
 
+## Building App Images Using `build`
+
+`pack build` enables app developers to create runnable app images from source code using buildpacks.
+
+```bash
+$ pack build <image-name>
 ```
-./pack build packs/myimage:mytag --path ./myapp
+
+### Example: Building Using the Default Builder Image
+
+In the following example, an app image is created from Node source code.
+
+```bash
+$ cd /path/to/node/app
+$ pack build my-app:my-tag
+
+# ... Detect, analyze and build output
+
+Successfully built 2452b4b1fce1
+Successfully tagged my-app:my-tag
 ```
 
-The above will create images on your local daemon. If you wish to export images directly to a docker registry, use the `--publish` flag.
+In this case, the default [builder](#working-with-builders-using-create-builder) is used, and an appropriate buildpack
+is automatically selected from the builder based on the app source code. To understand more about what builders are and
+how to create or use them, see the
+[Working with Builders Using `create-builder`](#working-with-builders-using-create-builder) section.
 
+To publish the produced image to an image registry, include the `--publish` flag (this is most useful
+for private registries):
```

Comment:
This is actually more useful for public registries (where stack authors could maintain a run image).